src/model/Column.ts

```diff
@@ -436,7 +442,18 @@ export default class Column extends AEventDispatcher {
    * @return {string}
    */
   rendererType(): string {
```

Comment:
since there is now a corresponding setter operation, rename this function to the corresponding getter pattern `getRendererType`